cmd/snapd-apparmor/snapd-apparmor

```diff
@@ -49,6 +49,11 @@ is_container_with_internal_policy() {
 	# shellcheck disable=SC3043,SC2039
 	local ns_name
 
+	# WSL needs to be detected explicitly
+	if [ $(systemd-detect-virt --container) = "wsl" ]; then
```

Comment:
I hope the output never includes spaces, but let's be on the safe side:
```suggestion
	if [ "$(systemd-detect-virt --container)" = "wsl" ]; then
```

Reply:
Seems legit.